easybuild/easyconfigs/b/bsoft/bsoft-1.8.8-goolf-1.5.14.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'MakeCp'
+
+name = 'bsoft'
+version = '1.8.8'
+
+homepage = 'http://lsbr.niams.nih.gov/bsoft/'
+description = """Bsoft is a collection of programs and a platform for development of software 
+for image and molecular processing in structural biology. Problems in structural biology 
+are approached with a highly modular design, allowing fast development of new algorithms 
+without the burden of issues such as file I/O. It provides an easily accessible interface, 
+a resource that can be and has been used in other packages."""
+
+toolchain = {'name': 'goolf', 'version': '1.5.14'}
+toolchainopts = {'openmp': True}
+
+source_urls = ['http://lsbr.niams.nih.gov/bsoft/']
+sources = ['%(name)s1_8_8.tar']
+
+prebuildopts = [("./bmake omp fftw=$EBROOTFFTW #")]
```

Comment:
ok, this doesn't work since `CmdCp` currently simply iterates over all source files, and figures out which commands to run for each of them

this doesn't work here, since it concerns a tarball being unpacked...